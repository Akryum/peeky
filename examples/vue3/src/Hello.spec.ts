// import { describe, test, expect } from '@peeky/test'
import { mount } from '@vue/test-utils'
import Hello from './Hello.vue'

describe('vue + peeky demo', () => {
  test('mount component', async () => {
    console.log(Hello)

    const wrapper = mount(Hello, {
      props: {
        count: 4,
      },
    })

    expect(wrapper.text()).toContain('4 x 2 = 8')

    expect(wrapper.html()).toMatchSnapshot()

    await wrapper.get('button').trigger('click')

    expect(wrapper.text()).toContain('4 x 3 = 12')

    await wrapper.get('button').trigger('click')

    expect(wrapper.text()).toContain('4 x 4 = 16')
  })
})
