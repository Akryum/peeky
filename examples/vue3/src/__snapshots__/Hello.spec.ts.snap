// Peeky snapshots v1.0

exports[`vue + peeky demo mount component 1`] = `
<pre>
  4 x 2 = 9
</pre>
<button>
  More
</button>
`;